src/components/DoDontCaution/index.css

```
@@ -48,7 +48,7 @@
 }
 
 .caution-icon > svg {
-  margin-top: 3px;
+  margin-top: 0.1875rem;
```

Should this not be:
```suggestion
  margin-top: 0.188rem;
```
In previous work we have rounded to 3 decimal places.